lib/prism/ripper_compat.rb

```diff
@@ -108,7 +108,8 @@ def parse
 
     # Visit a CallNode node.
     def visit_call_node(node)
-      if !node.message.match?(/^[[:alpha:]_]/) && node.opening_loc.nil? && node.arguments&.arguments&.length == 1
+      message = node.message
+      if message && !message.match?(/^[[:alpha:]_]/) && node.opening_loc.nil? && node.arguments && node.arguments.arguments && node.arguments.arguments.length == 1
```

Comment:
Steep's flow analysis doesn't follow `&.`, so I expanded it if that's okay. Otherwise I can wrangle a type assertion on a local variable.